MyXamarinApp.API/Controllers/EmployeeController.cs

```diff
@@ -21,21 +23,79 @@ public EmployeeController(IEmployeeRepository employeeRepository)
         public async Task<ActionResult<List<EmployeeModel>>> GetAllEmployees()
         {
             var employees = await _employeeRepository.GetAllEmployees();
-            return employees;
+            if (employees != null)
+            {
+                return Ok(employees);
+            }
+            else
+            {
+                return NotFound();
+            }
+        }
+
+        [HttpGet("{id}")]
```

Comment:
I am curious to know what happens if I send something like GET employee/abc ? Will it throw an exception, or will it return a 404 Not Found ?
I guess before making a call to GetEmployeeById we can check if guid is an empty guid